modules/javafx.graphics/src/main/java/javafx/css/CssMetaData.java

```diff
@@ -327,5 +328,23 @@ public int hashCode() {
             .append("}").toString();
     }
 
-
+    /**
+     * Utility method which combines {@code CssMetaData} items in one unmodifiable list with size equal
+     * to the number of items it holds.
+     *
+     * @param list the css metadata items, usually from the parent, must not be null
+     * @param items the additional items
+     * @return the unmodifiable list containing all of the items
+     *
+     * @since 22
+     */
+    public static List<CssMetaData<? extends Styleable, ?>> combine(
+        List<CssMetaData<? extends Styleable, ?>> list,
+        CssMetaData<? extends Styleable, ?>... items)
+    {
```

Comment:
The method is not actually making use of `CssMetaData`, so a more generic utility method would have worked as well:

         public static <T> List<T> combine(List<T> list, T... items)

It could be placed in some kind of utility class (and in fact, many libs offer such a method already).

Reply:
Yeah, perhaps even in List?  
JavaFX does not provide a common utility package, and I am trying to limit the scope by narrowing the accepted data type here.